test/test-frames/ELS_Elster-F96-Plus.xml

```diff
@@ -48,14 +48,14 @@
         <Function>Value during error state</Function>
         <StorageNumber>0</StorageNumber>
         <Unit>Power (W)</Unit>
-        <Value>144445223</Value>
+        <Value>DDDDEBBD</Value>
```

Comment:
This looks like a hexadecimal number. Should this be using some other encoding?

Reply:
This is the intention of this patch. Since this is a error value (look at the function), it's better to signalize this with a hex value than a unrealistic decimal.